packages/elvis-icons/src/store.ts

```diff
@@ -0,0 +1,39 @@
+interface IconData {
+  svg: string;
+}
+type IconRegistry = Record<string, IconData>;
+
+let _elviaGlobalIconsStore: IconRegistry = {};
+
+/**
+ * Add icons to the global icon store to make them available for use in the `<e-icon>/Icon` component.
+ *
+ * You can add multiple icons at once. The key of the input object is the icon name corresponding to the `name` prop in the `<e-icon>` component.
+ * While this can be any string, it is recommended to use the icon name as the key.
+ * @param icons - An object of icon names and their corresponding SVG data.
+ *
+ * @example
+ * ```ts
+ * import { addIcons } from '@elvis/icons';
+ *
+ * // import the icon data (SVG string)
+ * import { download } from '@elvia/elvis-assets-icons';
+ *
+ * addIcons({
+ *  download: { svg: download.getIcon() }
+ * });
+ * ```
+ */
```

Comment:
If I remember correctly, you said that the filled state of the color depends on the icon name. Should we tell the user to include the word "filled" in the icon name?